src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -478,9 +479,11 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             var connectors = obj["Connectors"].ToObject<IEnumerable<ConnectorModel>>(serializer);
 
             var info = new WorkspaceInfo(guid.ToString(), name, description, Dynamo.Models.RunType.Automatic);
-
-            //https://jira.autodesk.com/browse/QNTM-3872 
-            //Category should be set explicitly from custom node workspace.
```

Comment:
why this comment is removed?

Reply:
@ramramps I only removed Jira task link, the other comment is merged below.